### DISCUSSION TITLE
user-guide/edit-items

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#262] [文档评论区]
# user-guide/edit-items

Zotero 非官方中文维护小组

https://zotero-chinese.github.io/user-guide/edit-items

**Comment by cooltzy** [comment]
求教！当我拖入一个pdf时，条目偶尔只能显示文章的名字，其它信息均无，这时候我一般是去浏览器利用插件导入该PDF的条目信息。之前zotero6可以撤销检索元数据，然后通过手动创建父条目，输入DOI进行检索，不知道为什么zotero7没有撤销元数据的选项了。亦或者能有其它快捷方式通过DOI更新条目？请各位大佬不吝赐教！

**Reply by northword** [reply]
> 其它快捷方式通过DOI更新条目

https://zotero-chinese.com/user-guide/plugins/linter#%E6%A0%B9%E6%8D%AE%E6%A0%87%E8%AF%86%E7%AC%A6%E8%A1%A5%E5%85%A8%E6%9D%A1%E7%9B%AE%E7%9A%84%E6%9C%9F%E3%80%81%E5%8D%B7%E3%80%81%E9%A1%B5%E7%AD%89%E4%BF%A1%E6%81%AF

![Clip_2024-09-22_17-12-08](https://github.com/user-attachments/assets/e24b6aab-0c22-4849-be9b-a058c6af3da9)

**Comment by Athena-Psyer** [comment]
不确定条目的分类属性，但是在Google scholar上复制了apa的引用格式，能否有种方法可以直接把复制的格式复制进去然后zotero自动识别呢？
